.github/workflows/dart_ci.yml

```diff
@@ -68,6 +68,8 @@ jobs:
 
       - name: Create SBOM Release Asset
         uses: anchore/sbom-action@v0
+        # This fails if it runs more than once within a given build
+        if: matrix.sdk != '2.18.7'
```

Comment:
Do we even need to keep running CI on 2.18.7?

Reply:
Good question: I don't think so. I thought about ripping that out, but didn't wanna mess with it right now